README.md

```diff
@@ -83,6 +84,15 @@ BlazingMQ and see them in action.
 
 ---
 
+## Building
+
+[bin/build-ubuntu.sh](bin/build-ubuntu) and
+[bin/build-darwin.sh](bin/build-darwin) build BlazingMQ and its dependencies,
+respectively, on Ubuntu 22.04.2 LTS and Darwin 22.6.0. They can serve as a basis
+to build BlazingMQ on other systems.
+
```

Comment:
Can you also add instructions about how one can build and run unit tests here.

Reply:
> Can you also add instructions about how one can build and run unit tests here.

Done [here](https://github.com/bloomberg/blazingmq/pull/88).